motion_tracker/model/model.py

```diff
@@ -57,6 +64,8 @@ def make_model(img_edge_size, backend_id):
     start_box_mask_layer = start_box_mask
     for layer in pooling_layers:
         start_box_mask_layer = layer(start_box_mask_layer)
+    start_box_mask_layer = MaxPooling2D(pool_size=(3, 3), 
+                         dim_ordering=backend_id)(start_box_mask_layer)
 
```

Comment:
Hmmm.  

We'd like entry in the bounding box layer to line up directly on top of the entry representing those same pixels in the image.

But if the pooling isn't the same, than we're probably breaking that match-up.

For that matter, the padding should also change the alignment... and this extra layer of pooling may be offsetting the changes in padding.

I'm guessing we can keep the alignment right, and thus learn better if we replace `pooling_layers` with `shape_changing_layers`.

That would be something like 
`    shape_changing_layers = [layer for layer in reusable_img_featurizer.layers if (("pooling" in layer.name) or ("padding" in layer.name))]
`
Then line 71 becomes `for layer in shape_changing_layers` and we can get rid of the Pooling that we added on in 73-74